scripts/multisrc/lightnovelwp/template.ts

```diff
@@ -35,7 +37,7 @@ class LightNovelWPPlugin implements Plugin.PluginBase {
     this.site = metadata.sourceSite;
     const versionIncrements = metadata.options?.versionIncrements || 0;
     this.version = `1.0.${4 + versionIncrements}`;
```

Comment:
should update the template version

Reply:
it doesn't change anything for the other plugins, should i really update it for everyone?

Reply:
I guess no